sdcm/cluster_k8s/__init__.py

```diff
@@ -1923,12 +1923,38 @@ def add_scylla_cluster_value(self, path: str, element: Any):
         )
 
     @property
-    def scylla_config_map(self):
+    @contextlib.contextmanager
+    def scylla_config_map(self) -> dict:
+        exists = False
```

Comment:
This line (exists = False) could be defined on line 1935 instead of "pass".

Reply:
addressed